README.adoc

```diff
@@ -62,7 +62,10 @@ Minishift is an open-source project dedicated to developing and supporting Minis
 The code base is forked from the https://github.com/kubernetes/minikube[Minikube] project.
 
 Contributions, questions, and comments are all welcomed and encouraged!
-Minishift developers hang out on IRC in the #minishift channel on Freenode.
+Join the Minishift team on:
+
+- #minishift on https://freenode.net/[Freenode]
+- the https://lists.minishift.io/admin/lists/minishift.lists.minishift.io[Minishift Mailing List]
```

Comment:
I also like @thatdocslady 's suggestion i.e. 

You can reach the Minishift community by:

* Signing up to our [mailing list](https://lists.minishift.io/admin/lists/minishift.lists.minishift.io)
* Joining the #minishift channel on [Freenode IRC](https://freenode.net/)